RedBoxMachine.java

```diff
@@ -12,22 +12,24 @@
 public class RedBoxMachine
 {
    //Create an instance variable to hold all of the DVDs.
-
+   ArrayList<DVD> lib = new ArrayList<DVD>();
    /** the list of DVDs */
-   
+
    /** Constructs a Redbox Machine and fills it with DVDs
     *  Reads the file MovieList.txt so make sure that the
     *  file is in the same folder as the RedboxMachine.class
     *  file.
     */
-    
+
    public RedBoxMachine()
    {
       // Complete the constructor.
       // Leave this method. It will fill the machine with DVDs.
       fillMachine();
+      this.lib = lib;
```

Comment:
This is not necessary because you are not passing `lib` in through the constructor.